test/eventing-kafka.bash

```diff
@@ -76,6 +76,9 @@ function upstream_knative_eventing_kafka_broker_e2e {
 }
 
 function upstream_knative_eventing_kafka_broker_e2e_mesh() {
-   # TODO(pierdipi) Add tests from eventing-istio
+  # upstream_knative_eventing_e2e_mesh function in eventing.bash runs:
```

Comment:
It's a minor detail but I'd probably move this comment up to function `upstream_knative_eventing_kafka_broker_e2e` and remove calling this `upstream_knative_eventing_kafka_broker_e2e_mesh ` (it's a useless call)

Otherwise it looks good.

Reply:
+1, done